afs/media/js/views/components/workflows/analysis-areas-workflow/analysis-areas-annotation-step.js

```diff
@@ -386,7 +386,25 @@ define([
 
             self.savingTile(true);
             params.form.lockExternalStep('image-step', true);
-            self.savingMessage(`Saving Analysis Area ...`);
+
+            self.savingMessage(`This step is saving ...`);
+            const savingMessages = [
+                `Analysis Area Name and Classification`,
+                `Relationship between Analysis Area, Project and Parent Object (${self.physicalThingName()})`,
+                `It may take a while, so do not move away from this step.`
```

Comment:
Let's change the message here to "This may take a while, please do not move away from this step."